edx/edx-ora2
# [BD-5] [BB-2502] ORA v3 separate Schedule & Assessment tab

The following PR tries to improve the Studio view of the ORA Block by creating a new Schedule and assessment steps tab in Studio.

**JIRA tickets**: 
- [TNL-7272](https://openedx.atlassian.net/browse/TNL-7272)

**Discussions**: None

**Dependencies**: None

**Screenshots**:
![image](https://user-images.githubusercontent.com/8607745/83953671-a877bb80-a85f-11ea-8acb-9acedc7af480.png)

**Sandbox URL**: TBD - the sandbox is being provisioned.

**Merge deadline**: None 

**Testing Instructions**:
- Go to ORA(open response assessments) tab(the Peer Assessed Essays are ORA) in Studio and check the tab and the items there.

**Author notes and concerns**: None

**Reviewers**
